4_sexual_health.do

```diff
@@ -46,7 +46,10 @@
 			}
 			if inlist(country_name,"Thailand2019") {
 				global cp4 "cp4a cp4b cp4c cp4d cp4e cp4f cp4g cp4h cp4i cp4n cp4j cp4o"
-			}			
+			}
+			if inlist(country_name,"Tonga2019") {
+				global cp4 "cp4a cp4b cp4c cp4d cp4e cp4f cp4g cp4h cp4i cp4j "
```

Comment:
On page 116, I identified 9 methods of modern forms of contraception

Reply:
right, i also found the table. tabulation is as followed:
cp4a - female sterillization
cpb4 - male sterilization
cp4c - IUD
cp4d - injectables
cp4e - implants
cp4f - pills
cp4g - male condom
cp4h - female condom
cp4i - diapharagm
cp4j - jelly
the number mismatch you might point at is because in the data, jelly and diapharagm is seperated two variables instead one variable in the survey table